backend/src/main/java/com/redhat/cloud/notifications/db/Query.java

```diff
@@ -193,11 +178,6 @@ public Optional<Sort> getSort() {
         if (!sortFields.containsKey(lowerCaseSortColumn)) {
             throw new BadRequestException("Unknown sort field specified: " + sort.sortColumn);
         } else {
-            // NOTIF-674 Delete after confirming all deprecated fields are no longer used
-            if (deprecatedSortFields != null && deprecatedSortFields.contains(lowerCaseSortColumn)) {
-                Log.warnf("NOTIF-674 Using deprecated sort field: %s", lowerCaseSortColumn);
```

Comment:
Checked the logs and the sentry channel, the last occurence is April 4th

Reply:
I started writing a question exactly about that and got distracted by something else. You answered before I could post it 😄